test/integration/template_database_test.go

```diff
@@ -251,6 +251,60 @@ func TestDatabaseNodePortServiceRenders(t *testing.T) {
 	}
 }
 
+func TestDatabaseInvalidArchiveType(t *testing.T) {
+	// Path to the helm chart we will test
+	helmChartPath := testlib.DATABASE_HELM_CHART_PATH
+
+	options := &helm.Options{
+		SetValues: map[string]string{
+			"cloud.provider":                        "amazon",
+			"database.te.externalAccess.enabled":    "true",
+			"database.te.externalAccess.type":       "NodePort",
+			"database.te.externalAccess.internalIP": "true",
```

Comment:
not sure why they are needed, but when I remove them the test case fails with `Error: could not find template templates/service.yaml in chart`

Reply:
That's because you're resolving `templates/service.yaml` in the next line. You're basically just trying to render the templates and actually have no interest in the service. It would more sense to supply `templates/statefulset.yaml`, since that will always be present and is where the `archiveType` value is used (even though you're only checking that an error occurred and are not interested in the content).